### PR TITLE
mtl: add MicSel support

### PR DESCRIPTION
**Pull request by RanderWang**
Add MicSel config int mtl.toml

Prepare for MicSel support in SOF FW.

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@marcinszkudlinski any comments ?

**Review by marcinszkudlinski — CHANGES_REQUESTED** [review]

File: config/mtl.toml [hunk 1]
```diff
@@ -280,3 +280,22 @@ count = 9
 				20, 0, 0, 0, 12832, 20881000, 0, 0, 0, 20881, 0,
 				21, 0, 0, 0, 12832, 23431000, 0, 0, 0, 23431, 0,
 				22, 0, 0, 0, 12832, 30471000, 0, 0, 0, 30471, 0]
+
+	[[module.entry]]
+	name = "MICSEL"
+	uuid = "55A88ED5-3D18-46CA-88F1-0EE6EAE9930F"
```
[the remark this answers is not in the document]
Comment (RanderWang) [reply config/mtl.toml:286]: Do we need to reuse close source fw uuid ?  Driver gets uuid from manifest and doesn't have a copy in driver.

[the remark this answers is not in the document]
Comment (lgirdwood) [reply config/mtl.toml:286]: yes, we should use the same UUID.

[the remark this answers is not in the document]
Comment (RanderWang) [reply config/mtl.toml]: Thanks, updated UUID

**Review by marcinszkudlinski — APPROVED** [review]

**Pull request merged**